.github/workflows/ci.yml

```diff
@@ -10,6 +10,16 @@ jobs:
     permissions:
       packages: write
       contents: read
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_USER: dab
+          POSTGRES_PASSWORD: dabing
+          POSTGRES_ADMIN_PASSWORD: dabing
+          POSTGRES_DATABASE: dab_db
```

Comment:
@relrod you corrected this `POSTGRES_DB` in `make postgres`. Should it be changed here as well?

Reply:
good catch

Reply:
this was way harder than it needed to be

https://hub.docker.com/_/postgres

 - POSTGRES_PASSWORD
 - POSTGRES_USER
 - POSTGRES_DB

I guess we just kill off the admin password b/c don't need it.

Reply:
Actually do we even need the postgres stuff here? Tox will run it, and CI is using tox.

Reply:
Oh that's crazy. But it looks good to me. Delete the service here.